### ISSUE TITLE
[BUG][Oracle] panic during pull

### ISSUE THREAD

**Issue by youen** [CGI-FR/LINO#264]
Got this error while running a lino push command with lino-toolbox:1.4.0 on an oracle 19 database table.

the command
`lino pull ${SOURCE} --table ${TABLE} -l 0 | lino -v 3 push truncate --table ${TABLE} ${DESTINATION} -e error-ref-${TABLE}.json`

the stacktrace
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0xa01ab0]
goroutine 1 [running]:
github.com/cgi-fr/lino/internal/infra/push.OracleDialect.ConvertValue({}, {0xad6de0?, 0xc0006800c0?}, {{0xc00012a1f8, 0x8}, 0x0, {0x0, 0x0}})
	/workspace/internal/infra/push/datadestination_oracle.go:185 +0x70
github.com/cgi-fr/lino/internal/infra/push.(*SQLRowWriter).Write(0xc0004a2180, 0xc00068a1b0, 0xc0000b4140?)
	/workspace/internal/infra/push/datadestination_sql.go:345 +0x2ea
github.com/cgi-fr/lino/pkg/push.pushRow(0x42f7b0?, {0xcde350, 0xc0003c30e0}, {0xcdf300, 0xc0000b4140}, {0xcddb60, 0xc00005a4e0}, 0x0, {0xcd9eb8, 0xc000120980}, ...)
	/workspace/pkg/push/driver.go:224 +0x3df
github.com/cgi-fr/lino/pkg/push.Push({0xcde318, 0xc00005a510}, {0xcde350, 0xc0003c30e0}, {0xcddb60, 0xc00005a4e0}, 0x0?, 0x1f4, 0x0?, {0xcd7000, ...}, ...)
	/workspace/pkg/push/driver.go:70 +0x2e5
github.com/cgi-fr/lino/internal/app/push.NewCommand.func3(0xc0003cd500?, {0xc00013d280, 0x2, 0x8?})
	/workspace/internal/app/push/cli.go:155 +0x845
github.com/spf13/cobra.(*Command).execute(0xc0003cd500, {0xc00013d200, 0x8, 0x8})
	/home/vscode/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:987 +0xa91
github.com/spf13/cobra.(*Command).ExecuteC(0x1da1640)
	/home/vscode/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:1115 +0x425
github.com/spf13/cobra.(*Command).Execute(...)
	/home/vscode/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:1039
main.main()
	/workspace/cmd/lino/main.go:138 +0x25
```

was working with lino-toolbox:1.2.0 and 1.3.0

**Comment by adrienaury** [comment]
Seem to be an easy fix by testing if `descriptor.column` is null beforehand

```go
	if descriptor.column.Import() == "file" || descriptor.column.Import() == "blob" {
		//nolint:gocritic
		switch v := from.(type) {
		case []byte:
			return go_ora.Blob{Data: v, Valid: true}
		}
	}
```

**Comment by youen** [comment]
> Seem to be an easy fix by testing if `descriptor.column` is null beforehand
> 
> ```go
> 	if descriptor.column.Import() == "file" || descriptor.column.Import() == "blob" {
> 		//nolint:gocritic
> 		switch v := from.(type) {
> 		case []byte:
> 			return go_ora.Blob{Data: v, Valid: true}
> 		}
> 	}
> ```

We should cover that code by a unit test to avoid futur regressions.

**Comment by adrienaury** [comment]
Fixed in v2.7.1-beta1
